README.md

```diff
@@ -344,51 +347,56 @@ To get the best performances with img2dataset, using an efficient dns resolver i
 Follow [the official quick start](https://knot-resolver.readthedocs.io/en/stable/quickstart-install.html) or run this on ubuntu:
 
 install knot with
-```
+
+```bash
 wget https://secure.nic.cz/files/knot-resolver/knot-resolver-release.deb
 sudo dpkg -i knot-resolver-release.deb
 sudo apt update
 sudo apt install -y knot-resolver
 sudo sh -c 'echo `hostname -I` `hostname` >> /etc/hosts'
 sudo sh -c 'echo nameserver 127.0.0.1 > /etc/resolv.conf'
-udo systemctl stop systemd-resolved
+sudo systemctl stop systemd-resolved
 ```
 
 then start 4 instances with
-```
+
+```bash
 sudo systemctl start kresd@1.service
 sudo systemctl start kresd@2.service
 sudo systemctl start kresd@3.service
 sudo systemctl start kresd@4.service
 ```
 
 Check it works with
-```
+
+```bash
 dig @localhost google.com
 ```
 
 ### Setting up a bind9 resolver
 
 In order to keep the success rate high, it is necessary to use an efficient DNS resolver.
 I tried several options: systemd-resolved, dnsmaskq and bind9 and reached the conclusion that bind9 reaches the best performance for this use case.
-Here is how to set this up on ubuntu:
-```
+Here is how to set this up on Ubuntu:
+
+```bash
 sudo apt install bind9
 sudo vim /etc/bind/named.conf.options
 
-Add this in options:
-        recursive-clients 10000;
-        resolver-query-timeout 30000;
-        max-clients-per-query 10000;
-        max-cache-size 2000m;
+# Add this in options:
+#         recursive-clients 10000;
+#         resolver-query-timeout 30000;
+#         max-clients-per-query 10000;
+#         max-cache-size 2000m;
```

Comment:
Hey, I converted it into a different code block, so all can be copy-pasted and also it's clear they are different steps. Let me know if it looks good. Otherwise, I can undo it.